commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/ram/RamFileOutputStream.java

```diff
@@ -92,13 +90,7 @@ public void close() throws IOException {
         if (exception != null) {
             throw exception;
         }
-        try {
-            this.closed = true;
-            // Close the
-            this.file.endOutput();
-        } catch (final Exception e) {
-            throw new FileSystemException(e);
-        }
+        this.closed = true;
```

Comment:
This is a change in behavior that is likely to break some apps since `endOutput()` is no longer called. Needs more study...

Reply:
`endOutput` *will* be called because the only way to get a `RamFileOutputStream` is through `DefaultFileContent` which wraps the output stream and on close calls `endOutput`.

Reply:
Hi @rgoers,
Any thoughts as to whether or not this can break existing users?
TY!